winui/Avatar-view/customization.md

```diff
@@ -1,10 +1,176 @@
+
+Customize the border of `AvatarView` control by using the `BorderBrush` and `BorderThickness` properties.
+
+**BorderBrush**
+
+You can customize the border color of the AvatarView control by using the `BorderBrush` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView  ContentType="AvatarCharacter" 
+                          AvatarCharacter="Avatar16"
+                          AvatarSize="ExtraLarge"
+                          BorderBrush="Red">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.AvatarCharacter;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.AvatarCharacter = AvatarCharacter.Avatar16;
+avatarView.BorderBrush = new SolidColorBrush(Colors.Red);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderBrush](avatarview_images/winui_avatarview_borderbrush.png)
+
+**BorderThickness**
+
+You can customize the border thickness of the AvatarView control by using the `BorderThickness` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView   AvatarShape="Custom"
+                           AvatarSize="ExtraLarge"
+                           BorderBrush="Black" 
+                           BorderThickness="4">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarShape = AvatarShape.Custom;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.BorderThickness = new Thickness(4);
+avatarView.BorderBrush = new SolidColorBrush(Colors.Black);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderThickness](avatarview_images/winui_avatarview_borderthickness.png)
+
+## Background
+
+You can customize the background of the AvatarView control by using the `Background` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         InitialsType="DoubleCharacter"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+          
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom background](avatarview_images/winui_double_character_initialstype_avatarview.png)
+
+## Gradient background
+
+You can use the `LinearGradientBrush` or `RadialGradientBrush` for the Background property to specify a range of colors to apply the gradient background for the AvatarView, as shown in the following code example.
```

Comment:
Use the `LinearGradientBrush` or `RadialGradientBrush` for the Background property to specify a range of colors to apply the gradient background for the AvatarView as shown in the following code example.